src/changefile/getChangedPackages.ts

```diff
@@ -1,57 +1,96 @@
-import { ChangeFileInfo, ChangeInfoMultiple } from '../types/ChangeInfo';
-import { getChangePath } from '../paths';
-import {
-  getChanges,
-  getStagedChanges,
-  git,
-  fetchRemoteBranch,
-  parseRemoteBranch,
-  findPackageRoot,
-} from 'workspace-tools';
 import fs from 'fs-extra';
 import path from 'path';
 import minimatch from 'minimatch';
+import { ChangeFileInfo, ChangeInfoMultiple } from '../types/ChangeInfo';
+import { getChangePath } from '../paths';
+import { getChanges, getStagedChanges, git, fetchRemoteBranch, parseRemoteBranch } from 'workspace-tools';
 import { getScopedPackages } from '../monorepo/getScopedPackages';
 import { BeachballOptions } from '../types/BeachballOptions';
-import { PackageInfos } from '../types/PackageInfo';
+import { PackageInfos, PackageInfo } from '../types/PackageInfo';
+
+function getMatchingPackageInfo(
+  file: string,
+  cwd: string,
+  packageInfosByPath: { [packageAbsPosixPath: string]: PackageInfo }
+) {
+  cwd = path.posix.normalize(cwd);
```

Comment:
This makes me wonder if we need to add windows to our github action os matrix

Reply:
Yeah that could be a good idea. The annoying thing is I don't even know if this is needed, I'm kind of just guessing it might be...